.github/workflows/esp32-simtest.yaml

```diff
@@ -76,9 +89,14 @@ jobs:
           set -eu
           apt update
           DEBIAN_FRONTEND=noninteractive apt install -y -q \
-              doxygen erlang-base erlang-dialyzer \
+              doxygen erlang-base erlang-dev erlang-dialyzer erlang-eunit \
```

Comment:
Let's revert following changes in release-0.6. We are not using rebar3 and it has been fixed in main.